Source/Stationeers.Addons.Patcher/Core/Patchers/MonoPatcher.cs

```diff
@@ -123,23 +117,22 @@ public bool IsPatched()
             return false; // Not patched
         }
 
-        private void DiscoverInstallInstance()
+        /// <inheritdoc />
+        private void GetInstanceAssemblies(String installInstance)
         {
             // This is kind of verbose, might need to be rewritten in a more concise manner
-            if (File.Exists(Path.Combine(Environment.CurrentDirectory, Constants.GameResourcesDir, AssemblyDir, AssemblyName)))
+            String InstallResourcesDir = null;
```

Comment:
Please, use `var` or `string`. Uppercase `String` is only provided for the compiler and should not be used.

Reply:
+ local variables should be using lower case first letter. So: `installResourcesDir`.

Reply:
Resolved: https://github.com/Erdroy/Stationeers.Addons/pull/3/commits/be189895d13e95ed244c746dae0a87eeb55a0482